src/spdx_tools/spdx/validation/license_expression_validator.py

```diff
@@ -40,7 +40,7 @@ def validate_license_expression(
         )
 
     validation_messages = []
-    license_ref_ids: List[str] = [license_ref.license_id for license_ref in document.extracted_licensing_info]
+    license_ref_ids: List[str] = [license_ref.license_id.lower() for license_ref in document.extracted_licensing_info]
```

Comment:
going to `.lower()` feels wrong, as the upper case is the default for most licenses?

Reply:
Should I take any other approach to make licenseIds case insensitive.(we have couple of other methods other than `.lower()`)
 
Or keeping them case sensitive is just fine:(

Well, what are your thoughts?

Reply:
Sorry, I'm new to licenses and stuff.

Reply:
I think uppercase would be more consistent

Reply:
https://github.com/spdx/ntia-conformance-checker/issues/165#issuecomment-1872563211
@maxhbr we got this bug in ntia-conformance-checker and i was trying to fix it

Reply:
As per spdx documentation licenseId is not a case-sensitive. [link](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/#d2-case-sensitivity)